main.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```

Comment:
- [ ] Kindly add `#each` method in `MyList` class. In MyList implement a method #each that yields successive members of @list and uses the MyEnumerable module.
the code should resemble this: 
```
def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
```
- [ ] The initialize class should look like below:
```
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

Reply:
Implement the changes, thank you.